File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/models/ElectricitySource.java
package com.kendoui.spring.models;

public class ElectricitySource {
    private String source;
    private int percentage;
    private Boolean explode;

    public ElectricitySource(String source, int percentage, Boolean explode)
    {
        this.setSource(source);
        this.setPercentage(percentage);
        this.setExplode(explode);
    }

    public String getSource() {
        return source;
    }

    public void setSource(String source) {
        this.source = source;
    }

    public int getPercentage() {
        return percentage;
    }

    public void setPercentage(int percentage) {
        this.percentage = percentage;
    }

    public Boolean getExplode() {
        return explode;
    }

    public void setExplode(Boolean explode) {
        this.explode = explode;
    }
}

File: wrappers/mvc/demos/Kendo.Mvc.Examples/Models/Chart/ElectricitySource.cs
using System;
using System.Linq;

namespace Kendo.Mvc.Examples.Models
{
    public class ElectricitySource
    {
        public ElectricitySource()
        {
        }

        public ElectricitySource(string source, int percentage)
        {
            Source = source;
            Percentage = percentage;
        }

        public string Source { get; set; }
        public int Percentage { get; set; }
        public bool Explode { get; set; }
    }
}